conf/flight_plans/rotorcraft_guided_flightplan.xml

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
+
+<flight_plan alt="1.0" ground_alt="0" lat0="51.990634" lon0="4.376789" max_dist_from_home="20" name="Rotorcraft Optitrack (Delft)" security_height="0.3">
+  <header>
+
+#include "autopilot.h"
+
+/********************************************//**
+ *  ABI messages for exception triggers or added values for the guided flight plan
+ ***********************************************/
+
+// ABI for obstacle detection (In here it comes from the module colorfilter)
+#include "subsystems/abi.h"
+
+#ifndef FP_GUIDED_OBSTACLE_DETECTION_ID
+#define FP_GUIDED_OBSTACLE_DETECTION_ID ABI_BROADCAST    
+PRINT_CONFIG_VAR(FP_GUIDED_OBSTACLE_DETECTION_ID)
+#endif
+
+static abi_event obstacle_detection_ev;
+static void obstacle_detection_cb(uint8_t sender_id, float distance_obstacle, float heading_obstacle);
+float distance_obstacle_FP;
+static void obstacle_detection_cb(uint8_t UNUSED(sender_id), float distance_obstacle, float UNUSED(heading_obstacle))
+{
+  distance_obstacle_FP = distance_obstacle;
+}
```

Comment:
Ah yes, this is an better option. I did get many unused variable warnings so this will fix the problem. I do agree about that the proper solution is to automatically generate it, so I hope that will be possible